pandas/core/arrays/categorical.py

```diff
@@ -2175,7 +2175,8 @@ def _reduce(self, name, axis=0, **kwargs):
             raise TypeError(f"Categorical cannot perform the operation {name}")
         return func(**kwargs)
 
-    def min(self, numeric_only=None, **kwargs):
+    @deprecate_kwarg(old_arg_name="numeric_only", new_arg_name="skipna")
+    def min(self, skipna=True, **kwargs):
```

Comment:
do we need kwargs here?

Reply:
`_reduce` passes through kwargs, and they are further passed through to `min`

In theory, `np.min` has additional arguments that you could pass ... (but I don't think that will be used in practice). But when called from `Series`, we don't pass through any additional kwargs, so this can probably removed.

Reply:
@makbigc can you remove the `**kwargs` ?

Reply:
@makbigc can you just do this one?